ts/src/renderer/phaser/classes/PhaserRegion.ts

```diff
@@ -9,12 +9,23 @@ class PhaserRegion extends Phaser.GameObjects.Graphics {
 	) {
 		super(scene);
 
-		const stats = this.region._stats.default
+		// looking at whether to use _stats.default or _stats.currentBody
+		// appears the implementation of currentBody for regions is unfinished,
+		// but everything we should need for rendering is contained in default.
+		//
+		// I believe this is an issue unique to 'Region'
+		const stats = this.region._stats.default;
+
+		// const stats = this.region._stats;
 
 		// draw rectangle
 		const width = this.width = stats.width;
 		const height = this.height = stats.height;
-		this.fillStyle(0xFF0000, 0.4);
+		// Phaser wants a number for these
+		this.fillStyle(
+			Number(`0x${stats.inside.substring(1)}`) || 0xffffff,
+			stats.alpha / 100 || 0.4
```

Comment:
default value for `stats.alpha` should stay,  default for `stats.inside` should be removed. We should only emit `create-region` if there is a value for `.inside` on the `Region` entity.